web/themes/new_weather_theme/templates/partials/hourly-table.html.twig

```diff
@@ -60,15 +60,26 @@
               {% else %}
               <span class="text-base font-mono-xs"> {{ "N/A" | t }} </span>
               {% endif %}
-              {% if period.probabilityOfPrecipitation and period.probabilityOfPrecipitation > 1 %}
-              <p class="text-base text-ls-neg-3 font-family-mono font-mono-xs margin-top-05 margin-bottom-0"> {{period.probabilityOfPrecipitation}}% {{ "chance" | t }}<span class="usa-sr-only"> {{ "of precipitation" | t }}</span></p>
-              {% endif%}
             </div>
           </div>
           {% endif %}
         </td>
         {% endfor %}
       </tr>
+      <tr data-row-name="chance-precipitation">
+        <th scope="row" class="position-sticky left-0 font-family-mono text-primary-darker bg-primary-lightest font-mono-xs text-uppercase z-400">
+          {{"Chance of precipitation" | t}}
+        </th>
+        {% for period in hours %}
+        <td> 
+          {% if period.probabilityOfPrecipitation is not same as (null) %}
```

Comment:
This adds back the `0%` chance, right?

Reply:
That's right, my understanding is that `is not same as` is `!==` under the hood

Reply:
Yep. Okay, cool. I was asking because the old code didn't show precip chances below 1%, but if we're going to have a row dedicated to precip chance, it makes sense (to me, anyway) to always show it.

Reply:
That's my thinking as well – that 0% chance is actually meaningful data